README.md

```diff
@@ -115,3 +115,25 @@ Arguments for the rake task:
 
 See notes in the [README_work_zips.md](README_work_zips.md)
 
+## Related URL Field
+
+In this application, the `Related URL` field has been customized to allow Markdown text. With this change users can enter URLs like this:
+
+```markdown
+[Link Text](http://url_here.com) Explanatory text
+```
+
+If a user forgets to insert the markup and just adds a URL, that will be autolinked:
+
+```markdown
+http://url_here.com Explanatory text
+```
+
+HTML call also be entered in the field.
+
+In the help field for the entry there is a link to instructions on how to make links in Markdown: https://daringfireball.net/projects/markdown/syntax#link.
+
+The UI is also changed from a single line text entry to a mutli-line textarea:
+
+![screenshot-2018-2-7 new publication research database](https://user-images.githubusercontent.com/4324761/35922000-f1871a90-0be1-11e8-882f-da9eadb97a98.png)
+
```

Comment:
I can't see this image in the main view for the README.  Could you try checking that screenshot into the repo and linking to it from there?

Reply:
Oops I guess those github uploads only work where they first happen